seatunnel-api/src/main/java/org/apache/seatunnel/api/sink/multitablesink/MultiTableSink.java

```diff
@@ -164,4 +168,13 @@ public List<TablePath> getSinkTables() {
     public void setJobContext(JobContext jobContext) {
         sinks.values().forEach(sink -> sink.setJobContext(jobContext));
     }
+
+    @Override
+    public List<SchemaChangeType> supports() {
+        SeaTunnelSink firstSink = sinks.entrySet().iterator().next().getValue();
```

Comment:
The sink Action may include a variety of different kinds of Sink, if the first Sink that do not support SupportSchemaEvolutionSink and other support?

Reply:
For MultiTableSink it should only contain sinks of the same kind
cc @Hisoka-X

Reply:
> For MultiTableSink it should only contain sinks of the same kind

Right for now.